lib/Controller/ApiController.php

```diff
@@ -70,8 +72,8 @@ public function __construct(
 		$this->relatedService = $relatedService;
 		$this->configService = $configService;
 		try {
-			$this->circlesManager = \OC::$server->get(CirclesManager::class);
-		} catch (Exception $e) {
+			$this->circlesManager = OC::$server->get(CirclesManager::class);
```

Comment:
```suggestion
			$this->circlesManager = \OCP\Server::get(CirclesManager::class);
```

use public api

Reply:
wow, never heard of this, let me fix all call

Reply:
Carl added for 24 or 25.